src/main/java/de/rototor/pdfbox/graphics2d/PdfBoxGraphics2D.java

```diff
@@ -238,6 +239,19 @@ public void dispose() {
 	public void draw(Shape s) {
 		try {
 			contentStream.saveGraphicsState();
+
+			// Possibly set the alpha constant
+			if((composite != null) && (composite instanceof AlphaComposite))
```

Comment:
Same here, composite instanceof AlphaComposite will be false if composite is null